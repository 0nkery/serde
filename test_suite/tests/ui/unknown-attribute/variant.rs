#[macro_use]
extern crate serde_derive;

#[derive(Serialize)]
enum E {
    #[serde(abc = "xyz")]
    V,
}

fn main() {}
