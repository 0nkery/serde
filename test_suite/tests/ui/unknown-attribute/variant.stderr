error: unknown serde variant attribute `abc`
  --> $DIR/variant.rs:12:10
   |
12 | #[derive(Serialize)]
   |          ^^^^^^^^^

error: aborting due to previous error

